src/cores/stickers/StickerCard.js

```diff
@@ -18,8 +18,10 @@ function StickerCard({ sticker }) {
     return (
         
         <div key={sticker.id} className='bg-slate-400 p-4 rounded-md'>
+            {console.log(sticker.team)}
             <img src={sticker.img} alt='stickerImg' className='w-25' />
             <p className='text-sm'>Nombre: {sticker.playerName}</p>
+            <p className='text-sm'>Evento: </p>
```

Comment:
```suggestion
            {sticker.event?.eventName ? <p className='text-sm'>Evento: </p> : <></>}
```
Para poder mostrarlo cuando empiecen a enviartelo, es una mala practica, pero aja xD

Reply:
no hare eso, da sifilis.

Reply:
tienes buen criterio